lib/internal/Magento/Framework/App/FeedFactoryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedFactoryInterface
+{
+
+    const DEFAULT_FORMAT = 'rss';
```

Comment:
It's better to have constant name as 
`FORMAT_RSS` , but not `DEFAULT_FORMAT`

because if at some point of time some decide to change default format, he would not need to change the constant value

moreover, even from the readability point of view 
when you encounter in code `self::DEFAULT_FORMAT` you should go to the interface to see what's the format gonna be used.